cmd/clear.go

```diff
@@ -19,7 +19,7 @@ func clearCommand() *cobra.Command {
 
 	clearCmd.Flags().BoolP("verbose", "v", false, "Print paths that were cleared from the cache")
 	if err := viper.BindPFlag("verbose", clearCmd.Flags().Lookup("verbose")); err != nil {
-		cmdutil.ErrPrintf("%v\n", err)
+		log.Print(err)
```

Comment:
My concerns in https://github.com/puppetlabs/wash/pull/197#discussion_r274101636 still stand for this commit, unless it was accidentally introduced here?

Reply:
Oh right. I can add something. I think I had some theory of unifying logging, but I think I ended up just customizing logrus to be similar instead.